src/ClusterBootstrap/ctl.py

```diff
@@ -35,11 +35,25 @@
 from utils import walk_json, RestUtil
 
 
+def add_azure_params(config):
```

Comment:
@YinYangOfDao can you help review if this is appropriate? What else is needed to put allowlist in cloud init deployment pipeline?